clang/test/lab1/zakharov_artem/test.cpp

```diff
@@ -0,0 +1,101 @@
+// RUN: split-file %s %t
```

Comment:
Please, add some checks for
1. attempt of renaming non-existng variables
2. missing -plugin-arg-rename params errors